e2e/alphabet-card-detail.test.ts

```diff
@@ -12,15 +12,19 @@ const multiTap = async (tappable: ITappableAsync, numberOfTaps: number): Promise
     }
 };
 
+require('dotenv').config({ path: '.production.env' });
```

Comment:
Just curious, this is using a `.env` file rather than the approach of a `content.config.ts` we use in coscrad-frontend.  Should we keep to one approach or is this simpler in this case?